native/acter/src/api/news.rs

```diff
@@ -744,17 +747,100 @@ impl Space {
         })
     }
 
-    pub fn news_draft_with_builder(&self, content: NewsEntryBuilder) -> Result<NewsEntryDraft> {
-        if !self.is_joined() {
-            bail!("Unable to create news for spaces we are not part on");
-        }
-        Ok(NewsEntryDraft {
-            client: self.client.clone(),
-            room: self.inner.room.clone(),
-            content,
-            slides: vec![],
+    pub fn new_task_ref(
+        &self,
+        target_id: String,
+        room_id: Option<String>,
+        task_list: String,
+        action: String,
+    ) -> Result<RefDetails> {
+        let target_id = EventId::parse(target_id)?;
```

Comment:
Why are these on `Space` if they are not related to the same space?

Reply:
just globally is fine.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1369/commits/7bd1e46770feaf8f8525e410a7459c622caa256d